cli/main.rs

```diff
@@ -0,0 +1,67 @@
+mod runtime;
+
+use clap::{command, Parser, Subcommand};
+use deno_runtime::colors;
+use deno_runtime::deno_core;
+use deno_runtime::fmt_errors::format_js_error;
+
+use deno_core::error::JsError;
+
+use runtime::AnyError;
+
+use crate::runtime::run_js_module;
+
+#[derive(Parser)]
+#[command(author, version, about, long_about = None)]
+pub struct CliArgs {
+  #[command(subcommand)]
+  command: Commands,
+}
+
+#[derive(Subcommand)]
+pub enum Commands {
+  Run {
+    /// JavaScript file containing the Station Module to run
+    file: String,
+  },
+}
+
+#[tokio::main(flavor = "current_thread")]
+async fn main() {
+  #[cfg(windows)]
+  colors::enable_ansi(); // For Windows 10
+
+  match main_impl().await {
+    Ok(_) => (),
+    Err(err) => exit_with_error(err),
+  }
+}
+
+async fn main_impl() -> Result<(), AnyError> {
+  let cli_args = CliArgs::parse_from(std::env::args());
+  match cli_args.command {
+    Commands::Run { file } => {
+      println!("{} {file}", colors::green("EXECUTE"));
+      let main_module = deno_core::resolve_path(&file)?;
+      run_js_module(&main_module, &Default::default()).await?;
+      Ok(())
+    }
+  }
+}
+
+fn exit_with_error(error: AnyError) {
+  // Inspired by unwrap_or_exit<T> from https://github.com/denoland/deno/blob/main/cli/main.rs
```

Comment:
```suggestion
  // Inspired by unwrap_or_exit<T> from https://github.com/denoland/deno/blob/600fff79cdf5d52154344a0e3a8a523e1e21c3c1/cli/main.rs
```

Permalinks have higher chances or surviving

Reply:
> Permalinks have higher chances or surviving

Yeah, I usually use permalinks too.

In this PR, I prefer to link to the latest version of the Deno source code to make it easier to find the latest version. That's why I am linking to the entire file, not only to the line range for this particular method.

Reply:
Yes, but this entire file could be removed in the future! With a permalink you will always see at least what you saw when you wrote the code

Reply:
You are right. I reconsidered my original opinion and opened https://github.com/filecoin-station/zinnia/pull/16